databases/app-config-sync/fragment-feature-release-toggle-definitions.js

```diff
@@ -0,0 +1,37 @@
+{
+  channels: {
+    view: [ 'view-feature-release-toggle-definitions', 'view-config' ],
+    add: [ 'edit-feature-release-toggle-definitions', 'edit-config' ],
+    replace: [ 'edit-feature-release-toggle-definitions', 'edit-config' ],
+    remove: [ 'remove-feature-release-toggle-definitions', 'remove-config' ]
+  },
+  typeFilter: function(doc, oldDoc) {
+    // Example valid document IDs: "feature-release-toggle.foo-toggle", "feature-release-toggle.bar_1"
+    return /^feature-release-toggle\.[a-z0-9_-]+$/.test(doc._id);
+  },
+  propertyValidators: {
+    name: {
+      type: 'string',
+      required: true,
+      mustNotBeEmpty: true,
+      immutableWhenSet: true,
+      regexPattern: /^[a-z0-9_-]+$/
```

Comment:
Since this property's value is meant to exactly match the ID token at the end of the document's ID, I suggest updating the regular expression pattern to more closely match the convention established by @dkichler in the [paymentProcessorSettlement](https://github.com/Kashoo/kashoo-document-definitions/blob/22b085a3950928b8cd6ca7680a506461be4615e6/databases/business-sync/fragment-payment-processor-settlement.js#L44-L49) document type's `settlementId` property, wherein the regular expression matches the value from a match group extracted from the document ID.